extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisReceiver.scala

```diff
@@ -59,92 +60,103 @@ import com.amazonaws.services.kinesis.clientlibrary.lib.worker.Worker
  *                                 (InitialPositionInStream.TRIM_HORIZON) or
```

Comment:
@param initialPositionInStream seems to be missing

also, it appears to have moved below checkpointInterval which is a bit confusing.  i hit this while testing.

the @param docs may need to be re-ordered, as well